services/self-service/src/main/resources/mongo/general/mongo_roles.json

```diff
@@ -0,0 +1,100 @@
+[
+  {
+    "_id": "nbBillingReportFull",
+    "description": "View full billing report for all users",
+    "type": "BILLING",
+    "cloud": "AZURE",
```

Comment:
Do we need the **cloud** parameter here in general? Can we delete this parameter without any problems?